components/Cards/index.js

```diff
@@ -20,48 +20,40 @@
 
 const attach = document.querySelector('.cards-container')
 
-
-const test1 = [
-    [{
-        headline: "ES8: The Next Step in the Evolution of Javascript and What it Means For Your Projects",
-        authorPhoto: "./assets/sir.jpg",
-        authorName: "SIR RUFF'N'STUFF"
-    }],
-    [{
-        headline: "ES8: The Next Step in the Evolution of Javascript and What it Means For Your Projects",
-        authorPhoto: "./assets/sir.jpg",
-        authorName: "SIR RUFF'N'STUFF"
-    }],
-    [{
-        headline: "ES8: The Next Step in the Evolution of Javascript and What it Means For Your Projects",
-        authorPhoto: "./assets/sir.jpg",
-        authorName: "SIR RUFF'N'STUFF"
-    }]
-]
-
-test1.forEach(e => {
-    e.forEach(e => {
-        console.log(attach.appendChild(Cards(e)))
+axios.get(`https://lambda-times-backend.herokuapp.com/articles`)
```

Comment:
i like how you did all the logic in the .then instead of filling a state variable and then doing the logic. this would be faster and probably cause less errors kudos